src/foam/nanos/controller/ApplicationController.js

```diff
@@ -473,6 +483,17 @@ foam.CLASS({
         this.footerView_.removeAllChildren();
         this.footerView_.tag({ class: this.theme.footerView });
       }
-    }
+    },
+    {
+      name: 'updateDisplayWidth',
+      isMerged: true,
+      mergeDelay: 5000,
```

Comment:
let's reduce this delay a little. we made it big for testing to ensure it wasn't the cause of the slow resizing and it isn't. maybe reduce to 1000? or put it back to isFramed